app/views/admin/users/_member_index.html.erb

```diff
@@ -1,47 +1,74 @@
-<div class="crayons-card overflow-admin-main-layout-padding p-4 m:p-7">
+<div class="crayons-card overflow-admin-main-layout-padding p-4 xl:p-7">
   <header class="flex flex-col l:flex-row justify-content-between l:items-center ">
     <h1 class="crayons-title ml-2 m:ml-0">Members</h1>
     <%= render "admin/users/index/tabs" %>
   </header>
     <%= render "admin/users/index/controls" %>
 
-    <!-- Small screen data view start -->
-    <div class="block m:hidden fs-s">
-      <h2 class="member-data-heading fs-s py-2 -mx-3 px-3 color-base-60">Members</h2>
-      <ul class="list-none">
+    <!-- Responsive screen (breakpoints below XL) data view start -->
+    <div class="block xl:hidden fs-s">
+      <h2 class="member-data-heading fs-s py-2 -mx-4 px-6 color-base-60">Members</h2>
+      <ul class="list-none mx-2">
         <% @users.includes([:organization_memberships]).each do |user| %>
           <li class="py-4">
             <article>
-              <header class="flex">
-                <a href="<%= admin_user_path(user.id) %>">
-                  <img class="radius-full align-middle" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="<%= user.name %>" />
-                </a>
-                <div class="pl-3 flex-1 fs-s"><h3 class="fs-base"><%= link_to user.name, admin_user_path(user), class: "color-base-80" %></h3>
-                  <%= link_to "@#{user.username}", admin_user_path(user), class: "color-base-60" %>
+              <header class="flex items-center justify-between m:justify-start mb-2">
+                <div class="flex grow-1 mr-4 m:w-50 m:max-w-50">
+                  <a href="<%= admin_user_path(user.id) %>" class="mr-2">
+                    <img class="radius-full align-middle" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="<%= user.name %>" />
+                  </a>
+                  <div class="flex flex-col">
+                    <h3 class="fs-base"><%= link_to user.name, admin_user_path(user) %></h3>
+                    <div class="flex">
+                      <span class="color-base-60">@<%= user.username %></span>
+                      <span class="hidden m:block">
+                         &nbsp;
+                        <span class="c-indicator ml-2 crayons-hover-tooltip" data-tooltip="<%= cascading_high_level_roles(user) %>">
+                          <%= cascading_high_level_roles(user) %>
+                        </span>
+                      </span>
+                    </div>
+                  </div>
+                </div>
+                <div class="flex justify-between items-center grow-1">
+                  <div class="hidden m:block">
+                    <% if user.suspended? %>
```

Comment:
There is known repetition between layouts - refactoring into reusable partials is captured as a task in https://github.com/forem/forem/issues/16911, so please avert your eyes for now 😄